tests/unit_tests/test_filters.py

```diff
@@ -269,3 +269,20 @@ def test_energyfunc():
     np.testing.assert_allclose(f.energy, new_f.energy)
     np.testing.assert_allclose(f.y, new_f.y)
     assert f.interpolation == new_f.interpolation
+
+
+def test_tabular_from_energyfilter():
+    efilter = openmc.EnergyFilter([0.0, 10.0, 20.0, 25.0])
+    tab = efilter.get_tabular(values=np.array([5, 10, 10]))
+
+    assert tab.x.tolist() == [[0.0, 10.0], [10.0, 20.0], [20.0, 25.0]]
```

Comment:
This doesn't make sense to me. Shouldn't it just be a plain list?

Reply:
Currently ```tab.x``` is returning a np.array with a shape of (3, 2)
```python
tab.x
>>> array([[ 0., 10.],
           [10., 20.],
           [20., 25.]])
```

Reply:
```suggestion
    assert tab.x.tolist() == [0.0, 10.0, 20.0, 25.0]
```